src/index.js

```diff
@@ -221,23 +235,54 @@ OAuth.authorizeImplicitly = function(url, clientId, callback) {
  * @param  {String}   clientSecret The client secret
  * @param  {String}   username The username to log in.
  * @param  {String}   password The password to log in.
+ * @param  {String}   scopes The scopes with oauth
  * @param  {Function} callback Callback function, called when auth is done.
  * First arg is error object if any, second is an OAuth object
  */
 // TODO Support putting clientId/secret into basic auth header rather than body?
 // TODO According to the RFC spec, we shouldn't send client secret
-OAuth.authorizeWithPassword = function (url, clientId, clientSecret, username, password, callback) {
+OAuth.authorizeWithPassword = function (url, clientId, clientSecret, username, password, scopes, callback) {
+	var scope = scopes || '';
+
 	post(url, {
 		grant_type: 'password',
 		username: username,
 		password: password,
 		client_id: clientId,
-		client_secret: clientSecret
-	}, callback);
+		client_secret: clientSecret,
+		scope: scope
+	}, function(err, oauth) {
```

Comment:
This is unnecessary, pass in the callback function as-is, no need to wrap it with a function that just calls it anyways.